x-pack/plugins/security_solution/public/resolver/view/panels/index.tsx

```diff
@@ -39,7 +39,9 @@ const PanelContent = memo(function PanelContent() {
 
   const { timestamp } = useContext(SideEffectContext);
 
-  const { pushToQueryParams, queryParams } = useResolverQueryParams();
+  const queryParams = useSelector(selectors.breadcrumbParameters);
+
+  const pushToQueryParams = useReplaceBreadcrumbParameters();
```

Comment:
renamed `useResolverQueryParams` to this. it no longer returns the query params.